src/CppParser/Bootstrap/Bootstrap.cs

```diff
@@ -1,44 +1,46 @@
 using System;
+using System.Linq;
 using System.Collections.Generic;
 using System.IO;
 using CppSharp.AST;
 
-namespace CppSharp
+namespace CppSharp.Parser.Bootstrap
 {
     /// <summary>
     /// Generates parser bootstrap code.
     /// </summary>
     class Bootstrap : ILibrary
     {
-        static string GetSourceDirectory(string dir)
+        static string GetSourceDirectory (string dir)
```

Comment:
Please do not put spaces before opening parentheses, the project does not follow this style.